include/swift/Remote/CMemoryReader.h

```diff
@@ -40,8 +42,39 @@ namespace remote {
 class CMemoryReader final : public MemoryReader {
   MemoryReaderImpl Impl;
 
+  uint64_t ptrauthMask;
+
+  uint64_t getPtrauthMask() {
```

Comment:
Is this specific to ptrauth?  Or is it really just about masking off unrelated stuff from a pointer?

Reply:
Just ptrauth, it's specifically used for stripping signed pointers to get an address we can read remotely.